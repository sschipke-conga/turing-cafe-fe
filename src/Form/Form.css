form {
  display: flex;
  justify-content: space-between;
  width: 90%;
}

form input {
  width: 15%;
  padding: 5px;
}

form button {
  background: teal;
  color: white;
  width: 15%;
}

button:hover {
  cursor: pointer;
}