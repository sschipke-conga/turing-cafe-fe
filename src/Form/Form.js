import React, {Component} from 'react';
import './Form.css'

class Form extends Component {
  constructor() {
    super()
    this.state = {
      
    }
  }

  render() {

  }
}

export default Form