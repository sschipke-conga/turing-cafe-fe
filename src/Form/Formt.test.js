import React from 'react';
import { shallow } from 'enzyme';
import Form from './Form'

describe('Form', () => {
  let wrapper;
  const mockAddReservation = jest.fn()
  beforeEach(() => {
    wrapper = shallow(<Form addReservation={mockAddReservation}/>)
  })
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})