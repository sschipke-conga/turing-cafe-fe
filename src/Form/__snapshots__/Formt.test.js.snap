// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should match the snapshot 1`] = `
<form>
  <input
    id="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value=""
  />
  <input
    id="date"
    onChange={[Function]}
    placeholder="Date (mm/dd)"
    type="text"
    value=""
  />
  <input
    id="time"
    onChange={[Function]}
    placeholder="Time"
    type="text"
    value=""
  />
  <input
    id="number"
    onChange={[Function]}
    placeholder="Number of guests"
    type="number"
    value=""
  />
  <button
    onClick={[Function]}
    type="button"
  >
    Make Reservation
  </button>
</form>
`;
